docs/api-reference/test-utils/browser-test-driver.md

```diff
@@ -97,19 +97,16 @@ window.browserTestDriver_finish('Congratulations! All tests passed.');
 
 Notify the node script that the app has finished executing and the browser should be closed.
 
-### browserTestDriver_isHeadless()
+### browserTestDriver_isHeadless
 
 ```js
-window.browserTestDriver_isHeadless().then(isHeadless => {
-  if (isHeadless) {
-    console.log('Test is running in headless mode');
-  }
-});
+if (window.browserTestDriver_isHeadless) {
```

Comment:
Is this a function or a variable?

```
if (window.browserTestDriver_isHeadless) {
// vs
if (window.browserTestDriver_isHeadless()) {
```

Reply:
It is a function used as if it's a variable (puppeteer can only inject functions)

Reply:
Maybe add some comment in code explaining how that works? To me it looks like the return value will be a function thus always true.